doc/sphinx/proof-engine/ltac.rst

```diff
@@ -252,6 +208,51 @@ improving performance.
         - a tactic
         -
 
+:n:`ltac:(@ltac_expr)` can be used to indicate that the parenthesized item
+should be interpreted as a tactic and not as a term.  The constructs can also
+be used to pass parameters to OCaml code.  (While all
+of the :token:`syn_value`\s can appear at the beginning of an :token:`ltac_expr`,
+the others are not useful because they will not evaluate to tactics.)
+
+:n:`uconstr : ( @term )` can be used to build untyped terms.
+Terms built in |Ltac| are well-typed by default.  Building large
```

Comment:
Have you already mentioned `constr:` at this point?  If not, it seems like you should move the constr discussion above this, or insert it, because otherwise "by default" is confusing.

Reply:
The sentence is in the second paragraph after this table that mentions `constr:`.  Is that sufficient?

![image](https://user-images.githubusercontent.com/1253341/83958605-9595c480-a828-11ea-9324-cffe65c10005.png)

Reply:
I think we want a tiny bit more than that.  I think something like
```suggestion
:n:`uconstr : ( @term )` can be used to build untyped terms.
Terms built in |Ltac| are well-typed by default, using :n:`constr : ( @term )`.  Building large
```
would be sufficient.

Reply:
Though maybe it's fine as-is, and "default" means "when used as arguments to Ltac functions"

Reply:
I think I'll leave it as-is.  I think we don't have a good writeup on typechecking yet, so harder to make this understandable.  Nor do I know that much about typechecking, e.g. when it runs.  I'll leave a todo in the file, though.